examples/coco/README.md

```diff
@@ -1,52 +1,165 @@
+
+In the original paper, the U-Net is described as follows:
+
+<img alt="Unet" height="400" src="./coco/images/Unet.png" width="600"/>
+
+#### Transfer Learning
+
+For the encoder, we use a pre-trained model - MobileNetV2. That will help to learn robust features and reduce the number of trainable parameters. The decoder/upsampler is simply a series of upsample blocks. During the training process, only the decoder is being trained when the encoder weights are being frozen.
+
+## Category Prediction Model
+
+The model's task is to segment images consisting of two categories: `person` and `car`.  
+
+First, we evaluate our model on a dataset subset containing cars and person instances:
+
+| Dataset            |Mean IoU Person   | Mean IOU car     |
+| --------           |  --------        | -------          |
+| Category Model     | 0.309            |    0.262         |
+
+#### Cluster Analysis
+
+To qualitatively analyze the model's predictions on the different classes we utilize Tesnorleap's latent space. We select samples from different areas of the embedding space and use `fetch similar` to create unique clusters of similar samples.
+
+Among the clusters we got are:
+
+#### B&W cluster 
+
+We fetch a group of similar images that are mostly greyscale images as seen below:
+
+<img alt="BWcluster" height="300" src="./coco/images/b_w_cluster.png" width="400"/>
+
+This cluster not only captures the single channel B&W images, but also those images with a small variation in
+[hue](https://en.wikipedia.org/wiki/HSL_and_HSV).
+
+<img alt="BWclusterpoints" height="400" src="./coco/images/b_w_cluster_points.png" width="700"/>
+
+In fact, viewing the cluster in Tensorleap's `cluster analysis` shows that the vast majority of images (colored red) 3-channel RGB imagees.
+
+If we compare B&W images to RGB images we see that, on average, our loss on RGB images is lower:
+
+<img height="550" src="./coco/images/b_w_loss.png" width="700"/>
+
+
+#### 'Vehicle-like' clusters
+Our model latent space have multiple semantically meaningful vehicle clusters
+#### Bicycle cluster 
+
+<img height="400" src="./coco/images/Bicycle_cluster.png" width="400"/>
+
+#### Bus cluster
+
+<img height="400" src="./coco/images/Bus_cluster.png" width="400"/>
+
+Suprisingly, the attention map that highlights the features that defines this custer contain not only bus feature, bus also buildings, towers, and similar constructs:
+
+<img height="400" src="./coco/images/Bus_cluster_attention.png" width="400"/>
+
+
+## Vehicle Supercategory Model
+
+Our previous model tries to segment cars as a seperate class from truck & bus (which are labeled background). 
```

Comment:
What do you guys say about writing here a flow including the insights gained by Tensorleap, something like: "After finding a big issue we have in the vehicle class using Tensorleap's algorithms we have decided to create the entire vehicle SuperCategory... "